tcms_tenants/context_processors.py

```diff
@@ -0,0 +1,17 @@
+def tenant_navbar_processor(request):
+    """
+    Provide tenant name for display in navbar
+    """
+    tenant_name = request.tenant.schema_name
+    if request.tenant.organization:
+        tenant_name = request.tenant.organization
+    customized_logo_contents = """
+                    <li style="float:left">
+                    <a>
+                        <span id="tenant">
+                        """ + tenant_name + """
+                        </span>
+                    </a>
+                </li>
+    """
```

Comment:
It's better to keep HTML and programming code separated. Move the HTML snippet into a template file and use `render_to_string()`, see https://github.com/kiwitcms/Kiwi/blob/master/tcms/bugs/tests/test_models.py#L41.

You can pass the desired name as a context variable.